### ISSUE TITLE
Verion # incompatible with Kodi despite release numbers matching

### ISSUE THREAD

**Issue by MarvelousMari** [xbmc/peripheral.joystick#256]
tldr: suspect something similar to #184  has occured again for latest Kodi release.
`2023-01-16 09:20:33.634 T:7448    error <general>: Add-on 'peripheral.joystick' is not compatible with Kodi
2023-01-16 09:20:33.635 T:7448    debug <general>: CAddonMgr: DisabledReason for peripheral.joystick updated to 2
`

Updated kodi this morning
`*@pop-os:~$ kodi --version
Kodi Media Center 20.0 (20.0.0) Git:20230115-389e701cb9
Copyright (C) 2005-2021 Team Kodi - http://kodi.tv
`
Launched kodi, got a pop-op saying the add-on is not compatible with this verion of kodi, currently installed version of peripheral.joystick is 19.0.3, running the aeon-trao skin (not sure if that affects things)

restarting, forcing and auto-updating do not work, have not tried manually compiling and adding the latest release

some snippets from  'kodi --debug --logging=console' where i navigated to addons and then attempted to enabled the peripheraal.joystick add-on later cancled ctrl^c in terminal in a rush...
```
2023-01-16 09:20:12.615 T:7448    debug <general>: CAddonInfoBuilder::ParseXMLTypes: Binary addon found: screensaver.xbmc.builtin.dim
2023-01-16 09:20:12.616 T:7448    debug <general>: CAddonInfoBuilder::ParseXMLTypes: Binary addon found: peripheral.joystick
2023-01-16 09:20:12.618 T:7448    debug <general>: CAddonMgr::FindAddons: Addon 'metadata.common.fanart.tv' already present with version 3.6.3 at '/usr/share/kodi/addons/metadata.common.fanart.tv/' replaced with version 3.6.4 at '/home/mar/.kodi/addons/metadata.common.fanart.tv/'
2023-01-16 09:20:12.619 T:7448    debug <general>: CAddonMgr::FindAddons: Addon 'service.xbmc.versioncheck' already present with version 0.5.19+matrix.1 at '/usr/share/kodi/addons/service.xbmc.versioncheck/' replaced with version 0.5.25+matrix.1 at '/home/mar/.kodi/addons/service.xbmc.versioncheck/'
2023-01-16 09:20:12.620 T:7448  warning <general>: CAddonMgr::FindAddons: Addon 'game.controller.default' already present with higher version 1.0.35 at '/usr/share/kodi/addons/game.controller.default/' - other version 1.0.31 at '/home/mar/.kodi/addons/game.controller.default/' will be ignored
2023-01-16 09:20:12.620 T:7448    debug <general>: CAddonMgr::FindAddons: Addon 'metadata.tvshows.themoviedb.org.python' already present with version 1.6.0 at '/usr/share/kodi/addons/metadata.tvshows.themoviedb.org.python/' replaced with version 1.6.3 at '/home/mar/.kodi/addons/metadata.tvshows.themoviedb.org.python/'

```

```
2023-01-16 09:20:12.631 T:7448     info <general>: CAddonMgr::FindAddons: metadata.tvshows.themoviedb.org v3.5.14 installed
2023-01-16 09:20:12.631 T:7448     info <general>: CAddonMgr::FindAddons: metadata.tvshows.themoviedb.org.python v1.6.3 installed
2023-01-16 09:20:12.631 T:7448     info <general>: CAddonMgr::FindAddons: peripheral.joystick v19.0.3 installed
2023-01-16 09:20:12.631 T:7448     info <general>: CAddonMgr::FindAddons: plugin.audio.kxmxpxtx.bandcamp v0.4.1+matrix.1 installed
2023-01-16 09:20:12.631 T:7448     info <general>: CAddonMgr::FindAddons: plugin.program.autocompletion v2.1.2 installed
```

```
2023-01-16 09:20:12.924 T:7448     info <general>: GL: Enabling VSYNC
2023-01-16 09:20:12.930 T:7448     info <general>: load keymapping
2023-01-16 09:20:12.931 T:7448     info <general>: Loading special://xbmc/system/keymaps/appcommand.xml
2023-01-16 09:20:12.931 T:7448     info <general>: Loading special://xbmc/system/keymaps/customcontroller.AppleRemote.xml
2023-01-16 09:20:12.931 T:7448     info <general>: Loading special://xbmc/system/keymaps/customcontroller.Harmony.xml
2023-01-16 09:20:12.931 T:7448     info <general>: Loading special://xbmc/system/keymaps/customcontroller.SiriRemote.xml
2023-01-16 09:20:12.931 T:7448     info <general>: Loading special://xbmc/system/keymaps/gamepad.xml
2023-01-16 09:20:12.932 T:7448     info <general>: Loading special://xbmc/system/keymaps/joystick.xml
2023-01-16 09:20:12.932 T:7448     info <general>: Loading special://xbmc/system/keymaps/keyboard.xml
2023-01-16 09:20:12.933 T:7448     info <general>: Loading special://xbmc/system/keymaps/mouse.xml
2023-01-16 09:20:12.933 T:7448     info <general>: Loading special://xbmc/system/keymaps/remote.xml
2023-01-16 09:20:12.933 T:7448     info <general>: Loading special://xbmc/system/keymaps/touchscreen.xml
```

```
2023-01-16 09:20:18.218 T:7448    debug <general>: ------ Window Init (AddonBrowser.xml) ------
2023-01-16 09:20:18.218 T:7448     info <general>: Loading skin file: AddonBrowser.xml, load type: KEEP_IN_MEMORY
2023-01-16 09:20:18.222 T:7448    debug <general>: CGUIMediaWindow::GetDirectory (addons://)
2023-01-16 09:20:18.222 T:7448    debug <general>:   ParentPath = []
2023-01-16 09:20:18.222 T:7448    debug <general>: CAddonDatabase: SELECT repo.id FROM repo .. took 0 ms
2023-01-16 09:20:18.227 T:7448    debug <general>: CAddonDatabase: query  SELECT addons.*, repo.addonID AS repoID FROM addons JOIN addonlinkrepo ON addons.id=addonlinkrepo.idAddon JOIN repo ON repo.id=addonlinkrepo.idRepo WHERE addonlinkrepo.idRepo IN (1) ORDER BY repo.addonID, addons.addonID returned 724 rows in 4 ms
2023-01-16 09:20:18.259 T:7448    debug <general>: CAddonDatabase::GetAddons took 36 ms
2023-01-16 09:20:18.260 T:7448    debug <general>: ADDONS: repository.xbmc.org - 724 addon(s) loaded
2023-01-16 09:20:18.261 T:7448    debug <general>: CAddonRepos::BuildUpdateOrOutdatedList: Building update list from installed add-ons
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = audioencoder.kodi.builtin.aac / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.2
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = audioencoder.kodi.builtin.wma / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.2
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = context.embuary.info / Origin = repository.xbmc.org / Version = 2.0.0
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = game.controller.default / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.35
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = game.controller.keyboard / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.1.26
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = game.controller.mouse / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.19
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = game.controller.snes / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.34
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = inputstream.adaptive / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 20.3.1
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.album.universal / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.1.9
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.albums.theaudiodb.com / Origin = repository.xbmc.org / Version = 1.2.3
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.artists.theaudiodb.com / Origin = repository.xbmc.org / Version = 1.1.2
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.artists.universal / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 4.3.10
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.common.allmusic.com / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.2.2
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.common.fanart.tv / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.6.4
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.common.imdb.com / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.2.7
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.common.musicbrainz.org / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 2.2.4
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.common.theaudiodb.com / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 2.0.3
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.common.themoviedb.org / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.2.18
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.generic.albums / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.14
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.generic.artists / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.14
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.local / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.1
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.themoviedb.org.python / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 2.0.0
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.tvdb.com.python / Origin = repository.xbmc.org / Version = 1.0.3
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.tvshows.themoviedb.org / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.5.14
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = metadata.tvshows.themoviedb.org.python / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.6.3
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = peripheral.joystick / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 19.0.3
2023-01-16 09:20:18.261 T:7448    debug <general>: ADDONS: update check: addonID = plugin.audio.kxmxpxtx.bandcamp / Origin = repository.xbmc.org / Version = 0.4.1+matrix.1
```

```
2023-01-16 09:20:20.015 T:7505    debug <general>: CAddonDatabase::GetAddons took 21 ms
2023-01-16 09:20:20.016 T:7505    debug <general>: ADDONS: repository.xbmc.org - 724 addon(s) loaded
2023-01-16 09:20:20.017 T:7505    debug <general>: CAddonRepos::BuildAddonsWithUpdateList: Building combined addons-with-update map from installed add-ons
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = audioencoder.kodi.builtin.aac / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.2
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = audioencoder.kodi.builtin.wma / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.2
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = context.embuary.info / Origin = repository.xbmc.org / Version = 2.0.0
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = game.controller.default / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.35
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = game.controller.keyboard / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.1.26
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = game.controller.mouse / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.19
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = game.controller.snes / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.34
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = inputstream.adaptive / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 20.3.1
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.album.universal / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.1.9
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.albums.theaudiodb.com / Origin = repository.xbmc.org / Version = 1.2.3
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.artists.theaudiodb.com / Origin = repository.xbmc.org / Version = 1.1.2
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.artists.universal / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 4.3.10
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.common.allmusic.com / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.2.2
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.common.fanart.tv / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.6.4
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.common.imdb.com / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.2.7
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.common.musicbrainz.org / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 2.2.4
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.common.theaudiodb.com / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 2.0.3
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.common.themoviedb.org / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.2.18
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.generic.albums / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.14
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.generic.artists / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.14
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.local / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.0.1
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.themoviedb.org.python / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 2.0.0
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.tvdb.com.python / Origin = repository.xbmc.org / Version = 1.0.3
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.tvshows.themoviedb.org / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.5.14
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = metadata.tvshows.themoviedb.org.python / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 1.6.3
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = peripheral.joystick / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 19.0.3
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = plugin.audio.kxmxpxtx.bandcamp / Origin = repository.xbmc.org / Version = 0.4.1+matrix.1
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = plugin.program.autocompletion / Origin = repository.xbmc.org / Version = 2.1.2
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = plugin.video.esa / Origin = repository.xbmc.org / Version = 1.3.0+matrix.1
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = plugin.video.eso / Origin = repository.xbmc.org / Version = 1.3.0+matrix.1
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = plugin.video.nasa / Origin = repository.xbmc.org / Version = 3.0.3+matrix.1
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = plugin.video.nhlgcl / Origin = repository.xbmc.org / Version = 2022.11.29+matrix.1
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = plugin.video.themoviedb.helper / Origin = repository.xbmc.org / Version = 4.10.14
2023-01-16 09:20:20.017 T:7505    debug <general>: ADDONS: update check: addonID = repository.xbmc.org / Origin = b6a50484-93a0-4afb-a01c-8d17e059feda / Version = 3.3.1
```

```
2023-01-16 09:20:27.863 T:7448    debug <general>: HandleKey: return (0xf00d) pressed, window 10040, action is Select
2023-01-16 09:20:27.863 T:7448    debug <general>: SetItem - Addon with id peripheral.joystick not found locally.
2023-01-16 09:20:27.863 T:7448    debug <general>: ------ Window Init (DialogAddonInfo.xml) ------
2023-01-16 09:20:27.863 T:7448     info <general>: Loading skin file: DialogAddonInfo.xml, load type: KEEP_IN_MEMORY
2023-01-16 09:20:29.887 T:7448    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-01-16 09:20:29.887 T:7448    debug <general>: HandleKey: down (0xf081) pressed, window 10146, action is Down
2023-01-16 09:20:30.011 T:7448    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-01-16 09:20:30.178 T:7448    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-01-16 09:20:30.178 T:7448    debug <general>: HandleKey: down (0xf081) pressed, window 10146, action is Down
2023-01-16 09:20:30.229 T:7448    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-01-16 09:20:33.551 T:7448    debug <general>: Keyboard: scancode: 0x24, sym: 0x0d, unicode: 0x0d, modifier: 0x0
2023-01-16 09:20:33.634 T:7448    debug <general>: Keyboard: scancode: 0x24, sym: 0x0d, unicode: 0x00, modifier: 0x0
2023-01-16 09:20:33.634 T:7448    debug <general>: HandleKey: return (0xf00d) pressed, window 10146, action is Select
2023-01-16 09:20:33.634 T:7448    error <general>: Add-on 'peripheral.joystick' is not compatible with Kodi
2023-01-16 09:20:33.635 T:7448    debug <general>: CAddonMgr: DisabledReason for peripheral.joystick updated to 2
2023-01-16 09:20:33.640 T:7448    debug <general>: ------ Window Init (DialogNotification.xml) ------
2023-01-16 09:20:39.438 T:7448    debug <general>: Keyboard: scancode: 0x25, sym: 0x132, unicode: 0x00, modifier: 0x40
2023-01-16 09:20:39.438 T:7448    debug <general>: HandleKey: ctrl-leftctrl (0x1f0d0) pressed, window 10146, action is 
2023-01-16 09:20:39.605 T:7448    debug <general>: Keyboard: scancode: 0x85, sym: 0x137, unicode: 0x00, modifier: 0x40
2023-01-16 09:20:39.605 T:7448    debug <general>: HandleKey: ctrl-leftwindows (0x1f0d6) pressed, window 10146, action is 
2023-01-16 09:20:47.248 T:7448    debug <general>: ------ Window Deinit (DialogNotification.xml) ------
2023-01-16 09:20:56.378 T:7463    debug <general>: Thread JobWorker 139752982758976 terminating (autodelete)
2023-01-16 09:20:56.378 T:7460    debug <general>: Thread JobWorker 139753312908864 terminating (autodelete)
2023-01-16 09:20:56.378 T:7462    debug <general>: Thread JobWorker 139752991151680 terminating (autodelete)
2023-01-16 09:20:56.380 T:7496    debug <general>: Thread JobWorker 139751000634944 terminating (autodelete)
2023-01-16 09:21:47.621 T:7451    debug <general>: Sink changed
2023-01-16 09:21:52.628 T:7451    debug <general>: Sink changed
2023-01-16 09:23:21.978 T:7448    debug <general>: ------ Window Init (Pointer.xml) ------
2023-01-16 09:23:22.234 T:7448    debug <general>: ------ Window Deinit (Pointer.xml) ------
2023-01-16 09:23:22.267 T:7448    debug <general>: ------ Window Init (Pointer.xml) ------
2023-01-16 09:23:22.465 T:7448    debug <general>: ------ Window Deinit (Pointer.xml) ------
2023-01-16 09:23:24.060 T:7448    debug <general>: ------ Window Init (Pointer.xml) ------
2023-01-16 09:23:29.105 T:7448    debug <general>: ------ Window Deinit (Pointer.xml) ------
2023-01-16 09:23:47.110 T:7478    debug <general>: [ script.embuary.helper ] Start new fanart grabber process
2023-01-16 09:23:47.134 T:7478    debug <general>: GetArtistsByWhereJSON query: SELECT a1.*, art.art_id AS idArt, art.type AS artType, art.url AS artURL FROM (SELECT artist.idArtist, strArtist FROM artist  WHERE (EXISTS (SELECT 1 FROM song_artist WHERE song_artist.idArtist = artist.idArtist AND song_artist.idRole = 1) OR EXISTS (SELECT 1 FROM album_artist WHERE album_artist.idArtist = artist.idArtist)) AND (artist.strArtist != '') ORDER BY RANDOM() LIMIT 40) AS a1 LEFT JOIN art ON art.media_id = a1.idArtist AND art.media_type = 'artist' GROUP BY a1.idArtist, art.art_id ORDER BY a1.idArtist, arttype
2023-01-16 09:23:47.135 T:7478    debug <general>: GetArtistsByWhereJSON - query took 0 ms
2023-01-16 09:20:18.259 T:7448    debug <general>: CAddonDatabase::GetAddons took 36 ms
202^C
*@pop-os:~$ 2023-01-16 09:26:47.531 T:7448     info <general>: Quitting due to POSIX signal
```

**Comment by MarvelousMari** [comment]
I'm not certain where to find/if there is a .xml file the add-on uses for version tracking, or logging but more than happy to retry enabling the add-on and posting the correct log files. Thank you so much

**Comment by garbear** [comment]
Kodi v20 requires peripheral.joystick >= 20.0.0 (I just pushed a 20.1.3 release, should be available tomorrow). Did you upgrade from v19? Was peripheral.joystick not upgraded?

**Comment by MarvelousMari** [comment]
After updating peripheral.joystick this morning the issue is no longer present.

Yes, yesterday I upgraded from kodi 19.? to kodi Kodi Media Center 20.0 (20.0.0) Git:20230115-389e701cb9, peripheral.joystick was not upgraded at the time. peripheral.joystick was only upgraded/upgradable when the 20.1.3 was released. 
I never recieved a 20.0.0 release, checking my update history i can't find a record of 20.0.0 ever being made available, even though i succesfully updated other add-ons and repositories in the timeframe of the last two releases.
